src/main/logic/Bullets.h

```diff
@@ -12,6 +12,11 @@
 
 namespace gs {
 
+class Bullets;
```

Comment:
Probably don't need to forward declare the class here as the definition is just a few lines below it - can just typeset after the def